orcid-web/src/main/resources/freemarker/layout/hotjar.ftl

```diff
@@ -9,6 +9,11 @@
 	        r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
 	        a.appendChild(r);
 	    })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
-	</script> 
+	</script>
+	<script>
+	    if (orcidVar && orcidVar.orcidId) {
+	        window.hj('identify', orcidVar.orcidId, {});
+	    }
```

Comment:
@DanielPalafox Once the user logs out we need to send null as the identity according to this docs

https://help.hotjar.com/hc/en-us/articles/360033640653-Identify-API-Reference#handling_user_ids

Reply:
We also need to use the "realOrcid" here

Reply:
Done!